cypress/integration/HelpWithTechnologyPage.test.js

```diff
@@ -0,0 +1,34 @@
+describe("Help with technology page", () => {
+  beforeEach(() => {
+    cy.visit("/teachers/help-with-technology/");
+    cy.shouldHaveHeaderAndFooter();
+    cy.shouldHaveBreadcrumbs();
+  });
+
+  it("Shows the Help with technology page", () => {
+    cy.shouldHaveHeading(
+      "Get help with technology for remote education during coronavirus (COVID-19)"
+    );
+    cy.contains("Previous").should("exist");
+    cy.contains(
+      "Safeguarding and remote education during coronavirus (COVID-19)"
+    ).should("exist");
+  });
+
+  it('Links through to "Previous : Safeguarding and remote education during coronavirus (COVID-19)"', () => {
+    cy.contains(
+      "Safeguarding and remote education during coronavirus (COVID-19)"
+    )
+      .should((el) => {
+        expect(el).to.have.attr(
+          "href",
+          "/teachers/safeguarding-and-remote-teaching-during-coronavirus-covid-19/"
+        );
+      })
+      .click();
+    cy.location("pathname").should(
+      "equal",
+      "/teachers/safeguarding-and-remote-teaching-during-coronavirus-covid-19/"
+    );
+  });
```

Comment:
As with the previous comment, this test could be made reusable for the pagination component. 

This will save duplication for tests.

Reply:
Resolved